grml-debootstrap

```diff
@@ -1469,6 +1555,9 @@ prepare_vm() {
     else
       LOOP_PART="${LOOP_PART%p1}p3"
     fi
+  elif [ -n "$RPI" ]; then
+    export RPIFIRM_TARGET="/dev/mapper/$LOOP_PART" # '/dev/mapper/loop0p1'
```

Comment:
i have my doubts if this needs `export`

Reply:
It's used in a similar line for the EFI system partition a little bit above: `export EFI_TARGET="/dev/mapper/$LOOP_PART" # '/dev/mapper/loop0p1'` Maybe the export can be removed from both places?

Reply:
Removed export both from this and from the `EFI_TARGET` bit, seems that things still work right in my testing.